specification/search/data-plane/Microsoft.Azure.Search.Service/stable/2019-05-06/searchservice.json

```diff
@@ -5564,7 +5564,7 @@
       "required": true,
       "type": "string",
       "x-ms-skip-url-encoding": true,
-      "description": "The name of the Azure Search service.",
+      "description": "The name of the Azure Cognitive Search service.",
```

Comment:
I think we need to have consistency between the searchIndex.json and here...
There we called it "The name of the search service" and here we call it "The name of the Azure Cognitive Search service"....

I prefer what's in searchIndex.json ("The name of the search service") and let's apply that here as well...